go-controller/pkg/util/node_annotations.go

```diff
@@ -837,19 +855,19 @@ func IsOVNManagedNetwork(node *v1.Node, ip net.IP) (bool, error) {
 
 // GetNonOVNNetworkContainingIP attempts to find a non OVN managed network to host the argument IP
 func GetNonOVNNetworkContainingIP(node *v1.Node, ip net.IP) (string, error) {
-	networks, err := ParseNodeHostAddressesList(node)
+	networks, err := ParseNodeHostAddressesExcludeOVNManagedNetworks(node)
```

Comment:
I am wondering how are additional IP networks on br-ex other than the default network considered? Is that considered an OVN managed network in context of this functionality or is it not?

Reply:
There can be other IPs besides IPv4 and/or IPv6 address? Can you elaborate?

Reply:
I only exclude the primary network (which can be ipv4 and/or ipv6) - I am not aware of other networks added to br-ex.

Reply:
So it can have additional VIPs but those should be `/32` or `/128` addresses without a corresponding specific subnet route associated.

Then nothing prevents additional ip addresses to be added, after all, it's host networking not absolutely under our control but I guess then other stuff would not work either.

So I guess not a problem?

Reply:
If its `/32` or `/128` addresses we are good because an EIP cannot be assigned within that subnet.

Yeah - it would not work. I can call this out in docs here. Downstream, I have excluded secondary networks managed by OVN but its only in the docs which isnt a great user experience.